pandas/_libs/missing.pyx

```diff
@@ -40,6 +40,7 @@ cpdef bint checknull(object val):
      - NaT
      - np.datetime64 representation of NaT
      - np.timedelta64 representation of NaT
+     - NA
```

Comment:
this is an unrelated cleanup?

Reply:
Yeah I just noticed that was missing from the docstring so added it here